apps/src/templates/sectionProgressV2/InviteToV2ProgressModal.jsx

```diff
@@ -0,0 +1,185 @@
+import $ from 'jquery';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {connect} from 'react-redux';
+
+import Button from '@cdo/apps/componentLibrary/button/Button';
+import {Heading2, BodyTwoText} from '@cdo/apps/componentLibrary/typography';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import UserPreferences from '@cdo/apps/lib/util/UserPreferences';
+import AccessibleDialog from '@cdo/apps/templates/AccessibleDialog';
+import {default as LinkedButton} from '@cdo/apps/templates/Button';
+import {
+  setHasSeenProgressTableInvite,
+  setDateProgressTableInvitationDelayed,
+  setShowProgressTableV2,
+} from '@cdo/apps/templates/currentUserRedux';
+import i18n from '@cdo/locale';
+
+import styles from './progress-v2-invitation.module.scss';
+
+const newProgressViewGraphic = require('@cdo/static/teacherDashboard/progressOpenBetaAnnouncementGraphic.png');
+
+function InviteToV2ProgressModal({
+  sectionId,
+
+  // from redux
```

Comment:
Hmm, I have complicated thoughts about this kind of comment. On one hand I can see how it would be useful to the reader, on the other, these kinds of comments tend to get stale and unreliable very quickly. 

Do you see value in keeping this in the codebase long term or was this better for your initial iteration? 

I think my personal preference is to remove it, especially since there is already a list of props that are from redux in the `connect` function at the bottom of the file. But, I'll leave it up to you as an optional nit.

Reply:
Yeah, this is probably another place where it would be nice, as an org/team to get on the same page.  I see this `from redux` comment kinda a lot in our codebase.  I don't know if it is just because it is the "codebase I grew up with" or if there is a bigger meaning here, but I actually like the separation between "here's the props passsed in from another component" distinction.